### ISSUE TITLE
Extend getCommand API

### ISSUE THREAD

**Issue by ssire** [ssire/axel-forms#10]
Actually getCommand(host id, command name) returns the command with a given name hosted on a host element with a given id (e.g. a button id if the host element is a button).

There are some situations where it would be more convenient to get a command by name using the identifier of the editor. The reason is that the editor MUST have en identifier to be associated with a command through a data-target attribute, whereas it is not necessary to have an identifier on a host element of a command. 

So in summary we could refine the API to have two functions :

* getCommandByHost(host id, command name)
* getCommandByEditor(editor id, command name)

Note the principal use of this API is to programmatically trigger a command, e.g. `$axel.command.getCommand('editor', 'save').execute()`
